rasa_nlu/evaluate.py

```diff
@@ -469,7 +468,7 @@ def run_cv_evaluation(data, n_folds, nlu_config):
     logging.basicConfig(level=nlu_config['log_level'])
 
     if args.mode == "crossvalidation":
-        data = load_data(args.data)
+        data = DataLoader.load(args.data)
```

Comment:
I don't really like having a class that just consists of static methods (additionally we already have the `TrainingData` class). So from an api perspective I think 
```
from rasa_nlu import training_data

training_data.load()
``` 
is better

Reply:
👍 yeah I thought that it was a bit odd